pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools", "setuptools_scm"]
```

Comment:
```suggestion
requires = ["setuptools", "setuptools_scm", "wheel"]
```
Seems wheel will be needed too

Reply:
@esc this needs addressing?

Reply:
unlikely, `build` will build wheels fine without it, so I don't think it is strictly necessary

Reply:
ok, thanks for taking a look!